stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/model/AttachmentMetaData.kt

```diff
@@ -5,10 +5,10 @@ import android.net.Uri
 import com.getstream.sdk.chat.StreamFileUtil
 import com.getstream.sdk.chat.utils.Utils
 import io.getstream.chat.android.client.models.Attachment
-import io.getstream.chat.android.core.internal.InternalStreamChatApi
+import io.getstream.chat.android.core.ExperimentalStreamChatApi
 import java.io.File
 
-@InternalStreamChatApi
+@ExperimentalStreamChatApi
```

Comment:
😞 This class can be really confusing but at least it is marked with @ExperimentalStreamChatApi. We should probably revisit this implementation at some point